docs/source/dask.rst

```diff
@@ -72,6 +75,7 @@ Lastly, we can also manually start each worker individually (this is typically o
 .. code-block:: bash
 
     # server
+    # Note:
```

Comment:
Is this line supposed to be here or just a leftover?

Reply:
No.  Remove!